download/download.go

```diff
@@ -49,18 +47,6 @@ type Download struct {
 	Url string
 }
 
-// New initializes a new Download object which will download
-// the content from url into target.
-func New(url string, target Target) *Download {
-	return &Download{
-		HttpClient: new(http.Client),
-		Progress:   make(chan int),
-		Method:     "GET",
-		Url:        url,
-		Target:     target,
-	}
-}
-
```

Comment:
This method was unused and confusing because of the hard-coded `http.Client`. In practice this is always configured from `update.go` with the pre-set HTTP client.